Dashboard/Views/ScrutineeringView.axaml

```diff
@@ -76,87 +77,89 @@
-                                                                        <!-- Note:  We have to bind this way because regular bindings go to the carousels data context, so we need to get the relative data context, and cast it otherwise it thinks its an object-->
-                                                                        <Binding
-                                                                            Path="((vm:ScrutineeringViewModel)DataContext).SteeringAngle"
-                                                                            RelativeSource="{RelativeSource AncestorType=UserControl}"
-                                                                            StringFormat="F2" />
-                                                                        <Binding
-                                                                            Path="((vm:ScrutineeringViewModel)DataContext).AutonomousMissionIndicator"
-                                                                            RelativeSource="{RelativeSource AncestorType=UserControl}" />
-                                                                        <Binding
-                                                                            Path="((vm:ScrutineeringViewModel)DataContext).AutonomousSystemState"
-                                                                            RelativeSource="{RelativeSource AncestorType=UserControl}" />
-                                                                        <Binding
-                                                                            Path="((vm:ScrutineeringViewModel)DataContext).ServiceBrakeState"
-                                                                            RelativeSource="{RelativeSource AncestorType=UserControl}" />
-                                                                        <Binding
-                                                                            Path="((vm:ScrutineeringViewModel)DataContext).EmergencyBrakeState"
-                                                                            RelativeSource="{RelativeSource AncestorType=UserControl}" />
-                                                                    </MultiBinding.Bindings>
-                                                                </MultiBinding>
-                                                            </TextBlock.Text>
-                                                        </TextBlock>
-                                                    </ControlTemplate>
-                                                </ContentControl.Template>
-                                            </ContentControl>
-                                        </DataTemplate>
-                                    </ItemsControl.ItemTemplate>
-                                </ItemsControl>
+                <ScrollViewer VerticalScrollBarVisibility="Auto" AllowAutoHide="False">
```

Comment:
This overlaps the next button on smaller screens, if that matters...
![image](https://github.com/user-attachments/assets/523449a3-aee0-4790-b64c-6d74b890321c)